game-core/src/main/java/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -1508,8 +1508,17 @@ private static void placeUnits(
     final boolean clearFirst = clearFirstNewValue.getFirst();
     final String newValue = clearFirstNewValue.getSecond();
 
-    final boolean isValueTheSame =
-        newValue.equals(propertyAttachment.getRawPropertyString(propertyName));
+    final boolean isValueTheSame;
+    try {
+      isValueTheSame = newValue.equals(propertyAttachment.getRawPropertyString(propertyName));
+    } catch (final UnsupportedOperationException e) {
+      throw new UnsupportedOperationException(
```

Comment:
Codacy found an issue: [New exception is thrown in catch block, original stack trace may be lost](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5782710)